src/scripts/site.js

```diff
@@ -624,19 +608,16 @@ var AppInfo = {};
         }
 
         document.title = Globalize.translateDocument(document.title, "core");
-        var deps = ["apphost"];
 
         if (browser.tv && !browser.android) {
             console.log("Using system fonts with explicit sizes");
-            deps.push("systemFontsSizedCss");
+            require(["systemFontsSizedCss"]);
         } else {
             console.log("Using default fonts");
-            deps.push("systemFontsCss");
+            require(["systemFontsCss"]);
         }
 
-        deps.push("css!css/librarybrowser");
```

Comment:
why did you drop this?

Reply:
i didn't, i moved to the require function that comes right after that line. Webpack can't resolve dynamic modules passed as variables to require or define methods. So for certain cases, i merge and require them on the require calling.